src/command/src/lib.rs

```diff
@@ -92,6 +93,13 @@ impl CommandExt for Command {
     }
 
     fn result(&mut self) -> CommandResult {
+        let should_log = std::env::var("FLUVIO_CMD")
```

Comment:
can you put this using `once_cell::Lazy` so it can be only evaluated once?

Reply:
Sure thing